### PR TITLE
CAN-442 correct default values on create new product POST

### PR DESCRIPTION
**Pull request by alexcanopy**
## Description

Updating published default values on create product call - POST /product. Content-only change, no expected impact on canopy-api.

- late_fee_grace => 5
- product_color => #000000
- promo_min_pay_type => PERCENT_INTEREST

## Testing

execute a create product POST, do not pass the above 3 fields. Verify that product is created with the above default values.

- [X] Is QA required?

## Related issues
https://linear.app/canopy-inc/issue/CAN-442/bug-in-create-product-api-call-default-value-of-late-fees-is-incorrect

### CODE REVIEW

**Review by phorical — APPROVED** [review]

**Comment by phorical** [comment]
Double checked, for reference here's the default late_fee_grace: https://github.com/canopy1/canopy-api/blob/main/src/db/tables/products/products.sql#L103

File: schemas/FeePolicies.json [hunk 1]
```diff
@@ -3,9 +3,9 @@
   "properties": {
     "late_fee_grace": {
       "type": "string",
-      "description": "Defaults to 0 days. The amount of time after a payment is late after which you would like for a late fee to be incurred.",
+      "description": "The amount of time after a payment is late after which you would like for a late fee to be incurred.",
```
Comment (dcblair) [inline schemas/FeePolicies.json:6]: This is a little hard to follow, likely the two `afters` in the same sentence.
Not sure if this makes sense, but: `Once a payment is late, this is the period in which you would like to incur a late fee.`
@ps2-controller - Any recommendations?

File: types/go/CreateProductInput.go [hunk 2]
```diff
@@ -49,7 +49,7 @@ type DefaultAttributes struct {
 // The set of policies governing fees for the product.
 type FeePolicies struct {
 	DefaultSurchargeFeeStructure []DefaultSurchargeFeeStructure `json:"default_surcharge_fee_structure,omitempty"`// The transaction volume structure that dictates a transaction fee.
-	LateFeeGrace                 *string                        `json:"late_fee_grace,omitempty"`                 // Defaults to 0 days. The amount of time after a payment is late after which you would like; for a late fee to be incurred.
+	LateFeeGrace                 *string                        `json:"late_fee_grace,omitempty"`                 // The amount of time after a payment is late after which you would like for a late fee to; be incurred.
```
Comment (dcblair) [inline types/go/CreateProductInput.go:52]: See comment in `FeePolicies.json`

File: types/go/Product.go [hunk 3]
```diff
@@ -51,7 +51,7 @@ type DefaultAttributes struct {
 // The set of policies governing fees for the product.
 type FeePolicies struct {
 	DefaultSurchargeFeeStructure []DefaultSurchargeFeeStructure `json:"default_surcharge_fee_structure,omitempty"`// The transaction volume structure that dictates a transaction fee.
-	LateFeeGrace                 *string                        `json:"late_fee_grace,omitempty"`                 // Defaults to 0 days. The amount of time after a payment is late after which you would like; for a late fee to be incurred.
+	LateFeeGrace                 *string                        `json:"late_fee_grace,omitempty"`                 // The amount of time after a payment is late after which you would like for a late fee to; be incurred.
```
Comment (dcblair) [inline types/go/Product.go:54]: See comment in `FeePolicies.json`

File: types/ts/CreateProductInput.type.ts [hunk 4]
```diff
@@ -165,8 +165,8 @@ export interface FeePolicies {
      */
     default_surcharge_fee_structure?: DefaultSurchargeFeeStructure[];
     /**
-     * Defaults to 0 days. The amount of time after a payment is late after which you would like
-     * for a late fee to be incurred.
+     * The amount of time after a payment is late after which you would like for a late fee to
```
Comment (dcblair) [inline types/ts/CreateProductInput.type.ts:168]: See comment in `FeePolicies.json`

File: types/ts/Product.type.ts [hunk 5]
```diff
@@ -175,8 +175,8 @@ export interface FeePolicies {
      */
     default_surcharge_fee_structure?: DefaultSurchargeFeeStructure[];
     /**
-     * Defaults to 0 days. The amount of time after a payment is late after which you would like
-     * for a late fee to be incurred.
+     * The amount of time after a payment is late after which you would like for a late fee to
```
Comment (dcblair) [inline types/ts/Product.type.ts:178]: See comment in `FeePolicies.json`

**Review by dcblair — CHANGES_REQUESTED** [review]
Not sure if it needs changing, but I think updating the requested copy would really help with transparency.

File: schemas/FeePolicies.json (hunk 1 above)
Comment (alexcanopy) [reply schemas/FeePolicies.json:6]: Good point @dcblair. Doing some wordsmithing, what do you think about this:

`Amount of time between when a payment is due and when a late fee is assessed if no payments have been made.`

Comment (dcblair) [reply schemas/FeePolicies.json:6]: Wonderful!

**Review by dcblair — APPROVED** [review]
LGTM

File: schemas/FeePolicies.json (hunk 1 above)
Comment (dcblair) [reply schemas/FeePolicies.json:6]: Just need to update the comments in the other files.

Comment (alexcanopy) [reply schemas/FeePolicies.json:6]: Awesome possum thanks for the feedback. no worries about the comments in the typing those are auto-generated :)

Comment (dcblair) [reply schemas/FeePolicies.json]: That's right. My mistake.
